scripts/update_library_version.sh

```diff
@@ -198,18 +187,48 @@ export -f is_version_higher
 export -f update_library_version_pipfile
 export -f update_library_version_requirements
 
+# Skip double quotes that were not skipped in the libraries parameter
+libraries_json=$(echo "$libraries_json" | sed -E 's/(^|[^\\])"/\1\\"/g')
+
 # Find and update Pipfile files and requirements-elyra.txt files
 find "$directory" -type f \( -name "Pipfile" -o -name "Pipfile.gpu" -o -name "Pipfile.cpu" -o -name "requirements-elyra.txt" \) -exec bash -c '
   file=$0
-  lib=$1
-  new_ver=$2
-  include_paths_with=$3
-  exclude_paths_with=$4
-  lock_files=$5
-
-  case "$file" in
-    *Pipfile* ) update_library_version_pipfile "$file" "$lib" "$new_ver" "$include_paths_with" "$exclude_paths_with" "$lock_files" ;;
-    *requirements-elyra.txt* ) update_library_version_requirements "$file" "$lib" "$new_ver" "$include_paths_with" "$exclude_paths_with" ;;
-  esac
-' {} "$library_name" "$new_version" "$include_paths_with" "$exclude_paths_with" "$lock_files" \;
-
+  libraries_json=$1
+  include_paths_with=$2
+  exclude_paths_with=$3
+  lock_files=$4
+  files_to_lock_temp_file=$5
+
+  echo "'$libraries_json'" | jq -c ".[]" | while IFS= read -r lib_info; do
```

Comment:
Out of curiosity - is the both quote types intentional here - `"'`? Shouldn't be enough just the `"`?